run-in-docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+export JOB_SCHEDULER_DB_HOST=job-scheduler-database
+export JOB_SCHEDULER_DB_PORT=5432
+export JOB_SCHEDULER_DB_PASSWORD=test
```

Comment:
do all of those need to be set? (vs using defaults from config)

Reply:
They did at the time of writing this script. Now I've removed the unnecessary ones and left the password.